Pages/FoodCustomer/Cart.cshtml.cs

```diff
@@ -158,11 +170,13 @@ public IActionResult OnPostReserve()
             }
             catch (Exception)
```

Comment:
An exception is caught, but nothing is done. Maybe there should be some error data logging to a file?

Reply:
Should be